suites/nautilus/ansible/tier-0.yaml

```diff
@@ -0,0 +1,72 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/nautilus/ansible/tier-0.yaml
+# Smoke test cases for
+#    - Install RHCS4 with lvm scenario
+#
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      config:
+        is_production: true
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
+            fetch_directory: ~/fetch
+            ceph_origin: distro
+            ceph_stable_release: nautilus
```

Comment:
This configuration has no impact as we are using `ceph_origin: distro` hence can we remove it?